tests/publiccloud/storage_perf.pm

```diff
@@ -24,28 +24,29 @@ use constant IODEPTH => 4;
 sub run {
     my ($self) = @_;
     my $reg_code = get_var('SCC_REGCODE');
-    my $runtime = get_var('PUBLIC_CLOUD_FIO_RUNTIME',  300);
-    my $size    = get_var('PUBLIC_CLOUD_FIO_SSD_SIZE', 100);
-    my $url     = get_var('PUBLIC_CLOUD_PERF_DB_URI');
+    my $runtime   = get_var('PUBLIC_CLOUD_FIO_RUNTIME', 300);
+    my $disk_size = get_var('PUBLIC_CLOUD_FIO_SSD_SIZE');
+    my $disk_type = get_var('PUBLIC_CLOUD_SSD_TYPE');
+    my $url       = get_var('PUBLIC_CLOUD_PERF_DB_URI');
 
     my @scenario = (
         {
             name      => 'reference',
             rw        => 'randread',
             rwmixread => '100',
-            bs        => '8k'
+            bs        => '4k'
         },
         {
             name      => 'reallife',
             rw        => 'randrw',
             rwmixread => '65',
-            bs        => '8k'
+            bs        => '4k'
```

Comment:
what the reason to change it ?

Reply:
I've realized that most common value for storage performance is 4k and most common bs used by filesystems. Also, some disks in CVP have default physical block size of 4kb